ZygorGuidesViewer.lua

```diff
@@ -2831,86 +2957,37 @@ function me:UpdateSkin()
 
 	ZygorGuidesViewerFrame_Border_SectionTitle:SetTextColor(unpack(self.db.profile.skincolors.text))
 
-	ZygorGuidesViewerFrameScrollScrollBarScrollUpButton:SetNormalTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollUpButton,
-			SKINDIR .. "\\titlebuttons",
-			0.750,
-			0.875,
-			0.00,
-			0.25
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollUpButton:SetPushedTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollUpButton,
-			SKINDIR .. "\\titlebuttons",
-			0.750,
-			0.875,
-			0.25,
-			0.50
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollUpButton:SetDisabledTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollUpButton,
-			SKINDIR .. "\\titlebuttons",
-			0.750,
-			0.875,
-			0.75,
-			1.00
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollUpButton:SetHighlightTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollUpButton,
-			SKINDIR .. "\\titlebuttons",
-			0.750,
-			0.875,
-			0.50,
-			0.75
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollDownButton:SetNormalTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollDownButton,
-			SKINDIR .. "\\titlebuttons",
-			0.875,
-			1.000,
-			0.00,
-			0.25
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollDownButton:SetPushedTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollDownButton,
-			SKINDIR .. "\\titlebuttons",
-			0.875,
-			1.000,
-			0.25,
-			0.50
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollDownButton:SetDisabledTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollDownButton,
-			SKINDIR .. "\\titlebuttons",
-			0.875,
-			1.000,
-			0.75,
-			1.00
-		)
-	)
-	ZygorGuidesViewerFrameScrollScrollBarScrollDownButton:SetHighlightTexture(
-		CreateTextureWithCoords(
-			ZygorGuidesViewerFrameScrollScrollBarScrollDownButton,
-			SKINDIR .. "\\titlebuttons",
-			0.875,
-			1.000,
-			0.50,
-			0.75
-		)
-	)
+	-- Check which skin is active to set the correct scrollbar button textures
+	local choice = self.skin_choice[self.db.profile.skin]
+	if choice == "Modern" then

```

Comment:
Same thing here, can you put this code above the loadSkin one, so the defaults are already there, and we overwrite the values we want to in Z_SKIN_HOOKS